### PR TITLE
Grammar/style fixes for ITEM_SPAWN.md

### PR DESCRIPTION
**Pull request by actual-nh**
Grammar/style/clarification fixes for ITEM_SPAWN.md.

#### Summary

SUMMARY: None

#### Purpose of change

Grammar/style/clarification fixes for ITEM_SPAWN.md.

#### Describe alternatives you've considered

Not putting double spaces between sentences.

### CODE REVIEW

File: doc/ITEM_SPAWN.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ The procgen_id relates directly to a `relic_procgen_data` object's id. The `rule
 "charges": [10, 100]
 ```
 
-This will create 4 items, they can have different damage levels as the damage value is rolled separately for each of these items. Each item has charges (AKA ammo) in the range of 10 to 100 (inclusive); if the item needs a magazine before it can have charges, that will be taken care of for you. Using an array (which must have 2 entries) for charges/count/damage is equivalent to writing explicit min and max values. In other words `"count": [a,b]` is the same as `"count-min": a, "count-max": b`.
+This will create 4 items; they can have different damage levels as the damage value is rolled separately for each of these items.  Each item has charges (AKA ammo) in the range of 10 to 100 (inclusive); if the item needs a magazine before it can have charges, that will be taken care of for you.  Using an array (which must have 2 entries) for charges/count/damage is equivalent to writing explicit min and max values.  In other words, `"count": [a,b]` is the same as `"count-min": a, "count-max": b`.
 
 The container is checked and the item is put inside the container, and the charges of the item are capped/increased to match the size of the container.
```
Comment (actual-nh) [inline doc/ITEM_SPAWN.md:122]: I do not understand what "increased to match the size of the container" means here.

File: doc/ITEM_SPAWN.md [hunk 2]
```diff
@@ -244,20 +244,20 @@ Instead of a full JSON object, one can also write a JSON array. The default subt
...
 
 #### SUS
 
-When adding items to item groups, attempt to locate or create **SUS item groups.** SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given piece of storage furniture. SUS stands for "specific use storage." One of the aims of organizing item groups into specific use storage is to promote reusable tables that can be maintained and extended.
+When adding items to item groups, attempt to locate or create **SUS item groups.**  SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given piece of storage furniture.  SUS stands for "specific use storage."  One of the aims of organizing item groups into SUS groups is to promote reusable tables that can be maintained and extended.
 
-You can find SUS item groups at /data/json/itemgroups/SUS
+You can find existing SUS item groups at [/data/json/itemgroups/SUS](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/itemgroups/SUS/).
```
Comment (actual-nh) [inline doc/ITEM_SPAWN.md:263]: Should a note be made that, even if not using SUS groups, one should place item groups in data/json/itemgroups if they aren't only usable in a single file? Putting them there makes them much easier to find!

**Comment by kevingranade** [comment]
As this is a markdown document, it doesn't need to follow the rule for two spaces after a period.
That rule is necessary when text is displayed within the game itself, since the game only outputs fixed-width text.

File: doc/ITEM_SPAWN.md (hunk 1 above)
Comment (kevingranade) [reply doc/ITEM_SPAWN.md:122]: This occurs as per the note about charges above
``charges`: Setting only min and not max will make the game calculate the max charges based on container or ammo/magazine capacity.``
Setting just a mininum charges value means the number of charges will conform to the container.

**Pull request merged**